src/App.jsx

```diff
@@ -2,34 +2,40 @@ import React, { Component } from 'react';
 import './App.scss';
 import Banner from './components/Banner';
 import Header from './layout/Header';
+import Card from './card components/Card';
 
 class App extends Component {
   constructor(){
     super()
       this.state = {
         movies: [],
-        comics: []
+        comics: [],
+        isLoading: true
       }
   }
-
-  componentDidMount() {
-    fetch('https://gist.githubusercontent.com/MaxBSilver/bc1994851ccc283835a07735b0034201/raw/ce0cae47d5efaddffe35141aa2fe0d02f894e17f/marvel-movies.json')
+  
+  componentDidMount = () => {
```

Comment:
I think this is what's throwing the error for not having the information on page load. Maybe we can try changing it to a componentWillMount and see if that doesn't throw any errors.